jekyll/_cci2_ja/getting-started.adoc

```diff
@@ -47,190 +44,139 @@ CircleCI の `config.yml` オプションの全リストを確認したい場合
-{% include two-up.html title="4. おめでとうございます 🎉" content=content imageURL=congrats-first-green-pipeline imageAlt="Green Pipeline Build" %}
+{% include two-up.html title="5. おめでとうございます🎉" content=content imageURL=congrats-first-green-pipeline imageAlt="Passing pipeline" %}
 ++++
 --
 
-// Step 1 Gitlab
-
-[.tab.quickstart1.GitLab]
+[.tab.quickstart1.Set_up_project]
 --
 ++++
 {% capture content %}
-In the CircleCI web app, select <a  href="https://app.circleci.com/projects">Projects</a> from the sidebar. Click <strong>Create Project</strong>, then select the repository you would like to connect from the dropdown.
-Don’t see these options? 左上にある Orb Selector を使って正しい組織を見つけます。
+CircleCI ウェブアプリで、サイドバーから <a  href="https://app.circleci.com/projects">Projects</a> を選択します。リポジトリを検索し、<strong>Set Up Project</strong>をクリックします。
+リポジトリが見つかりませんか？左上の「Org Selector」を使用して、正しい組織を見つけます。
 {% endcapture %}
 
 {%- capture select-project -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/create-project.png
+  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/select-project.png
 {%- endcapture -%}
 
-{% include two-up.html title="1. Create a project" content=content imageURL=select-project imageAlt="Select Projects" %}
-
+{% include two-up.html title="1. プロジェクトを選択する" content=content imageURL=select-project imageAlt="Select Projects" %}
 
 {% capture content %}
-In the “Select your <a class="no-external-icon" href="https://circleci.com/docs/config-start/">config.yml</a> file” modal, select <b>Fast</b>, then give your project a name, and click <b>Create Project</b>.
+<a class="no-external-icon" href="https://circleci.com/docs/ja/config-start/">“config.yml</a> file” ファイルを選択してください。モーダルで <b>Fast</b>を選択し、<b>Set Up Project</b>をクリックします。 Hello World サンプル設定ファイルを選択します。
 {% endcapture %}
 
 {%- capture select-config -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/select-config-gitlab.png
+  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/select-config.png
 {%- endcapture -%}
 
-{% include two-up.html title="2. Select a config.yml" content=content imageURL=select-config imageAlt="Choose Config" %}
-
+{% include two-up.html title="2. config.yml を選択する" content=content imageURL=select-config imageAlt="Choose Config" %}
 
 {% capture content %}
-You are now in the <a class="no-external-icon" href="https://circleci.com/docs/config-editor/#getting-started-with-the-circleci-config-editor">CircleCI config editor</a>, pre-populated with a sample <code>config.yml</code> file. You can swap out this config for an alternative by clicking <strong>Change: Hello World</strong>.
+サンプル <code>config.yml</code> ファイルがあらかじめ入力された <a class="no-external-icon" href="https://circleci.com/docs/config-editor/#getting-started-with-the-circleci-config-editor">CircleCI config editor</a>に入ります。 そして<b>Click Commit and Run.</b> をクリックします。
+<br>
+<br>
+これにより、リポジトリのルート、<code>circleci-project-setup</code><code>という新しいブランチに <code>.circleci/config.yml</code> ファイルが作成されます。
```

Comment:
```suggestion
これにより、リポジトリのルート、<code>circleci-project-setup</code>という新しいブランチに <code>.circleci/config.yml</code> ファイルが作成されます。
```
There is one `<code>` element that doesn't have a closing tag, maybe the opening tag should be removed as well?